hedera-node/src/main/java/com/hedera/services/store/models/Topic.java

```
@@ -163,4 +176,76 @@ public boolean isNew() {
 	public void setNew(final boolean aNew) {
 		isNew = aNew;
 	}
+
+	public byte[] getRunningHash() {
+		return (runningHash != null) ? runningHash : new byte[RUNNING_HASH_BYTE_ARRAY_SIZE];
+	}
+	
+	public void setRunningHash(final byte[] runningHash) {
+		this.runningHash = runningHash;
+	}
+
+	/**
+	 * Updates the {@link Topic#runningHash} and the {@link Topic#sequenceNumber} properties of the topic.
+	 * Those changes have to be explicitly persisted
+	 * via {@link com.hedera.services.store.TopicStore#persistTopic(Topic)}.
+	 *
+	 * @param payer
+	 * 		- the account ID of the submitting member
+	 * @param message
+	 * 		- the actual message as bytes
+	 * @param topicID
+	 * 		- the ID of the topic
+	 * @param consensusTime
+	 * 		- consensus timestamp of the transaction
+	 * @return
+	 */
+	public ResponseCodeEnum updateRunningHashAndSeqNo(
```

We should not be using `ResponseCodeEnum`'s but rather throw an exception